data-loader/data/v2/products.csv

```diff
@@ -1197,10 +1204,14 @@ COVID-19 vaccine [viral vector],,,Parenteral,Suspension,AZD1222 [ChAdOX1-S] (Ast
 COVID-19 vaccine [viral vector],,,Parenteral,Suspension,AZD1222 [ChAdOX1-S] (AstraZeneca),1 x 10^11 vp per mL,5mL,Vial,10 doses,,,,933f3f00,3ab9c2c3e,ca6b457e2,8ea7a1bb7,a60328624,1d2ae3ed7,99cbab488,b9948cd18,ac6cbab3c,,,,2479831,,,,,,,,,,,,,,,,,,,,
 COVID-19 vaccine [viral vector],,,Parenteral,Suspension,,1 x 10^11 vp per mL,2.5mL,Vial,5 doses,,,,933f3f00,3ab9c2c3e,ca6b457e2,8ea7a1bb7,74d966d9e,4bdc32026,62a23619b,617a76ef9,14ec334e4,,,,2479831,,,,,,,,,,,,,,,,,,,,
 COVID-19 vaccine [viral vector],,,Parenteral,Suspension,Gam-COVID-Vac [Sputnik V] (Gamaleya),2 x 10^11 vp per mL,3mL,Vial,5 doses,,,,933f3f00,3ab9c2c3e,ca6b457e2,8ea7a1bb7,85d44350b,8465f8970,54c566d6b,8c1615989,4bacb5cbd,,,,2479831,,,,,,,,,,,,,,,,,,,,
-COVID-19 vaccine [mRNA-based],,,Parenteral,Suspension,BNT162b2 [Tozinameran] (Pfizer/BioNTech),0.1mg per mL,0.45mL,Vial,6 doses,,,,933f3f00,11c672231,a52316e87,235c18bc5,d4b6cb716,3fd9b240c,54c8891b4,5ec96b214,ada637fc6,,,,2468231,,,47407191000116100,47407241000116100,,,,,,,,,,,,,,,,
-COVID-19 vaccine [mRNA-based],,,Parenteral,Suspension,mRNA-1273 (Moderna),0.2mg per mL,5mL,Vial,11 doses,,,,933f3f00,11c672231,a52316e87,235c18bc5,2f99e94d1,cc578c833,765b31d17,be5ac1c73,78a1ad526,,,,2468231,,,,,,,,,,,,,,,,,,,,
-COVID-19 vaccine [mRNA-based],,,Parenteral,Suspension,mRNA-1273 (Moderna),0.2mg per mL,7.5mL,Vial,15 doses,,,,933f3f00,11c672231,a52316e87,235c18bc5,2f99e94d1,cc578c833,bbc37dc67,3694a9911,c56b7b834,,,,2468231,,,,,,,,,,,,,,,,,,,,
-COVID-19 vaccine [inactivated virus],,,Parenteral,Suspension,BBIBP-CorV (Sinopharm Beijing),,0.5mL,Pre-filled syringe,1 dose,,,,933f3f00,b1ab0664f,f3664828e,fcaed62ec,7b4668493,,a6ea839b7,149c11856,1f495d5c3,,,,,,,,,,,,,,,,,,,,,,,,
+COVID-19 vaccine [mRNA-based],Tozinameran,,Parenteral,Suspension,BNT162b2 [Tris/Sucrose Presentation] (Pfizer/BioNTech) Ready To Use,30mcg per 0.3mL dose,2.25mL,Vial,6 doses,,,,933f3f00,11c672231,a52316e87,235c18bc5,6ka0a68rq,zg10c351j,95b89q2y5,5ec96b214,31y89s2a5,,,,,,,,,,,,,,,,,,,,,,,,

```

Comment:
These vaccines will ultimately go in the other spreadsheet so not soo worried about them...